lib/generators/templates/atmosphere.rb

```diff
@@ -19,12 +19,6 @@
   #   main_app: [ AdditionalResourceViewsToShow ]
   # }
 
-  # PDP class for defining which Appliance Types user is able to start in
-  # development, production mode and which Appliance Types user is able to
-  # manage.
-  #
-  # config.at_pdp_class
```

Comment:
If this option is not used anymore how other projects like plgrid or exp in ismop will be working?